warehouse/query-core/src/test/java/datawave/query/util/regex/RegexTrieTest.java

```diff
@@ -0,0 +1,58 @@
+package datawave.query.util.regex;
+
+import org.apache.log4j.Level;
+import org.apache.log4j.Logger;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.regex.Pattern;
+
+public class RegexTrieTest {
+    private static final Logger log = Logger.getLogger(RegexTrieTest.class);
+
+    @BeforeClass
+    public static void setUpClass() throws Exception {
+        Logger.getRootLogger().setLevel(Level.OFF);
```

Comment:
There is a lot of direct logger level manipulation here. Is there a reason this can't go in the log4j.properties in the test/resources folder for this module?